src/plots/barPlot.ts

```diff
@@ -113,6 +120,56 @@ export class Bar<X, Y> extends XYPlot<X, Y> {
     return this;
   }
 
+  public barEnd(): Plots.TransformableAccessorScaleBinding<X, number>;
+  public barEnd(end: number | Accessor<number>): this;
+  public barEnd(end: X | Accessor<X>, scale: Scale<X, number>): this;
```

Comment:
barEnd should use the same scale as `.x` right? We should follow RectanglePlot's `.x2` here (it doesn't accept a scale)